template-parts/content-gallery.php

```diff
@@ -8,7 +8,11 @@
  */
 
 $gallery                = get_post_gallery( get_the_ID(), false );
-$gallery_attachment_ids = explode( ',', $gallery['ids'] );
+
+if ( isset( $gallery['ids'] ) ) {
```

Comment:
Better if we do like this:
`$gallery_attachment_ids = isset( $gallery['ids'] ) ? explode( ',', $gallery['ids'] ) : array();`

Reply:
Done @asheshmagar